pkg/tablebackup/tablebackup.go

```diff
@@ -24,10 +24,12 @@ import (
 )
 
 const (
+	// TODO: move those constants somewhere else
+	DeltasDirName      = "deltas"
```

Comment:
Do we really need to export those constants?

Reply:
it'll be used in the next PR, but I'll revert it as for now